auth/middleware.go

```diff
@@ -4,8 +4,11 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
+	"time"
 
+	jwt "github.com/appleboy/gin-jwt/v2"
```

Comment:
Just a quick heads up, I was checking https://jwt.io/ and it seems the underlying library being used by `gin-jwt` doesn't do all checks that JWT provides.
The website has some alternatives but then I don't know about using Gin with them.

Reply:
Here's the current checks: https://github.com/appleboy/gin-jwt/blob/master/auth_jwt.go#L373-L386. We can use the `IdentityHandler` and/or `Authorizator` for additional checks.